src/test/java/racingcar/winner/model/CarTest.java

```diff
@@ -0,0 +1,49 @@
+package racingcar.winner.model;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class CarTest {
+
+    @Test
+    void move_자동차가_전진한다() {
+        Car car = new Car(new CarName("test"));
+        Car movedCar = car.tryMove(new MovableStrategy() {
+            @Override
+            public boolean canMove() {
+                return true;
+            }
+        });
+
+        Assertions.assertThat(movedCar.getMoveResult().getMoveCount()).isEqualTo(1);
+        Assertions.assertThat(car.getMoveResult()).isNotEqualTo(movedCar.getMoveResult());
+    }
+
+    @Test
+    void move_자동차가_멈춘다() {
+        Car car = new Car(new CarName("test"));
+        Car movedCar = car.tryMove(new MovableStrategy() {
+            @Override
+            public boolean canMove() {
+                return false;
+            }
+        });
+
+        org.junit.jupiter.api.Assertions.assertAll(
+                () -> {
+                    Assertions.assertThat(movedCar.getMoveResult().getMoveCount()).isEqualTo(0);
+                },
+                () -> {
+                    Assertions.assertThat(car.getMoveResult()).isEqualTo(movedCar.getMoveResult());
+                });
+
```

Comment:
`Assertions#assertAll` 👍 
적용해 볼 만한 곳은 많은데 여기만 적용된 점은 조금 아쉽네요.
static import를 사용하면 좀 더 깔끔할 것 같아요. 

```suggestion
        assertAll(
                () -> assertThat(movedCar.getMoveResult().getMoveCount()).isEqualTo(0),
                () -> assertThat(car.getMoveResult()).isEqualTo(movedCar.getMoveResult())
        );

```